examples/python3/useful/header.py

```diff
@@ -17,7 +17,7 @@
 import os
 
 
-def addHeader(parentNode, multithreading=False, inverse=False, isConstrained=False, isContact=False):
+def addHeader(parentNode, multithreading=False, inverse=False, isConstrained=False, isContact=False, params=None):
```

Comment:
"params" is a too vague name, saying nothing on the what "params" there is in it. Makes me wonder if its purpose is not to act like a **kwargs